keyboards/fortitude60/rev1/rev1.h

```diff
@@ -0,0 +1,68 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "fortitude60.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+//void promicro_bootloader_jmp(bool program);
+
+#ifndef FLIP_HALF
```

Comment:
I'm not sure you need the FLIP_HALF, as .... I don't think you can accidently solder the wrong side. 

The thumb cluster only goes one way, after all.